perun-db/postgres.sql

```diff
@@ -1550,6 +1550,58 @@ create table groups_to_register (
     constraint grpreg_group_fk foreign key (group_id) references groups(id) on delete cascade
 );
 
+-- CONSENTS_DEFINITIONS - consents definitions for facilities, services and attributes
+create table consents_definitions (
+	id integer not null,
+	name varchar not null,
+	decription varchar,
+	created_at timestamp default statement_timestamp() not null,
+	created_by varchar default user not null,
+	modified_at timestamp default statement_timestamp() not null,
+	modified_by varchar default user not null,
+	created_by_uid integer,
+	modified_by_uid integer,
+	constraint con_def_pk primary key(id)
+);
+
+-- CONSENTS - relation between users and consent definitions
+create table consents (
+	user_id integer not null,		   --identifier of user
+	consent_def_id integer not null,   --identifier of consent definition
+	status varchar not null,
+	createdDate timestamp not null
+	constraint consents_pk primary key (user_id,consent_def_id),
+  	constraint consents_user_fk foreign key (user_id) references users(id),
+  	constraint consent_cons_def_fk foreign key (consent_def_id) references consents_definitions(id),
+  	constraint consents_stat_chk check (status in ('GRANTED','REVOKED','EXPIRED'))
+);
+
+-- FACILITY_CONSENT - relation between facilities and consent definitions
+create table facility_consent (
```

Comment:
I think this would be a little more accurate, what do you think? Same for other tables.
```suggestion
create table facilities_consent_definitions (
```

Reply:
You're right, I edited the names of these tables.